### DISCUSSION TITLE
Dynamic properties PHP 8.2

### DISCUSSION THREAD

**Discussion by ju5t** [ohdearapp/ohdear-php-sdk#38] [Q&A]
We're seeing this in our logs.

> Creation of dynamic property OhDear\PhpSdk\Resources\CronCheck::$createdAt is deprecated in .../vendor/ohdearapp/ohdear-php-sdk/src/Resources/ApiResource.php on line 29

The issue comes from:

```php
protected function fill(): void
{
    foreach ($this->attributes as $key => $value) {
        $key = $this->camelCase($key);

        $this->{$key} = $value;
    }
}
```

I'm a bit reluctant to PR a fix for it as I'm not 100% sure of the intention of this code and what I should expect.

**Comment by freekmurze** [comment]
This has been fixed in the latest release 👍

**Reply by ju5t** [reply]
Thanks!
